From d386b9cfa69eb68619e90e40a2dece1811cd2e27 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Tue, 22 Dec 2020 09:30:11 +0800
Subject: [PATCH] Revert "staging: ion: remove __GFP_NOWARN when use low order
 gfp flags"

This reverts commit d2805d7fff5ba1db3accd5ef010f9fdbb1925f08.

Change-Id: Id70bd22428df247703d3e4c466fe450faf680d03
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 drivers/staging/android/ion/ion_system_heap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/android/ion/ion_system_heap.c b/drivers/staging/android/ion/ion_system_heap.c
index a147e6c4bc4c..06c3e6c9c7ae 100644
--- a/drivers/staging/android/ion/ion_system_heap.c
+++ b/drivers/staging/android/ion/ion_system_heap.c
@@ -20,7 +20,7 @@
 
 static gfp_t high_order_gfp_flags = (GFP_HIGHUSER | __GFP_ZERO | __GFP_NOWARN |
 				     __GFP_NORETRY) & ~__GFP_RECLAIM;
-static gfp_t low_order_gfp_flags  = GFP_HIGHUSER | __GFP_ZERO;
+static gfp_t low_order_gfp_flags  = GFP_HIGHUSER | __GFP_ZERO | __GFP_NOWARN;
 static const unsigned int orders[] = {8, 4, 0};
 
 static int order_to_index(unsigned int order)
-- 
2.35.3

